bundle/manifests/mcg-osd-deployer.clusterserviceversion.yaml

```diff
@@ -36,6 +37,60 @@ spec:
     spec:
       clusterPermissions:
       - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
```

Comment:
@v-harihar do we need cluster level permission for services, deployments, deployments/finalizers, clusterversions, clusterversions/finalizers